inference-engine/samples/benchmark_app/main.cpp

```diff
@@ -117,8 +118,14 @@ T getMedianValue(const std::vector<T> &vec) {
 int main(int argc, char *argv[]) {
     std::shared_ptr<StatisticsReport> statistics;
     try {
+        Core ie;
         ExecutableNetwork exeNetwork;
 
+#ifdef USE_REMOTE_MEM
+        RemoteHelper remoteHelper;
```

Comment:
Can we get the device_name="VPUX"?

 If yes, we can put the the init code block into "if(device == "VPUX")" like "CPU" or "GPU" below.

Reply:
Done. Move the initialization to the branch of the 'VPUX'.